core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -108,8 +109,15 @@ public KafkaStore(SchemaRegistryConfig config,
         JaasUtils.isZkSecurityEnabled());
     this.brokerSeq = zkUtils.getAllBrokersInCluster();
 
-    this.bootstrapBrokers = KafkaStore.getBrokerEndpoints(
-            JavaConversions.seqAsJavaList(this.brokerSeq));
+    List<String> bootstrapServersConfig = config.getList(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG);
+    List<String> endpoints;
+    if (bootstrapServersConfig.isEmpty()) {
+      endpoints = brokersToEndpoints(JavaConversions.seqAsJavaList(this.brokerSeq));
+    } else {
+      endpoints = config.getList(SchemaRegistryConfig.KAFKASTORE_BOOTSTRAP_SERVERS_CONFIG);
+    }
+    this.bootstrapBrokers = filterBrokerEndpoints(endpoints);
```

Comment:
Great catch, @ewencp. I'm conflicted on this. I totally agree that an exception should be thrown in the bootstrap config case. However, you're right that soon all endpoints will be supported. I think changing the code to throw an exception is different enough than the current code that I'll leave what we have now and just wait for all endpoints to be supported.